jwt.go

```diff
@@ -175,7 +175,8 @@ func rSplit(s string, sep string, n int) []string {
 		for i, part := range parts {
 
 			if (i-1) < n {
-				if tempStr == "" {
+				//add condition on string length as well.
```

Comment:
This additional check is not necessary because it passes the empty string comparison it means it has length 0 already.

Reply:
Don't have time to address this change. Feel free to close this merge request.